core/app/models/workarea/catalog/product_placeholder_image.rb

```diff
@@ -44,6 +44,10 @@ def create_from_pipeline!
       def placeholder?
         true
       end
+      
+      def image_inverse_aspect_ratio
+        1 / image_aspect_ratio.to_f
```

Comment:
should we ever have to worry about `#image_aspect_ratio` being `nil` and resulting in a divide by 0 error?

Reply:
Good call, might as well return 0 if image_inverse_aspect_ratio is 0 or nil